source/custom/niscope_service.custom.cpp

```diff
@@ -634,16 +546,16 @@ ::grpc::Status NiScopeService::ReadMeasurement(::grpc::ServerContext* context, c
 
 ::grpc::Status NiScopeService::ConvertApiErrorStatusForViSession(google::protobuf::int32 status, ViSession vi)
 {
-    ViStatus error_code {};
-    std::string description(nidevice_grpc::kMaxGrpcErrorDescriptionSize, '\0');
-    // Try first to get the most recent error with a dynamic message.
-    library_->GetError(vi, &error_code, nidevice_grpc::kMaxGrpcErrorDescriptionSize, &description[0]);
-    if (error_code != status) {
-        // Since another thread has changed the status, fall back to the static message lookup.
-        description.assign(nidevice_grpc::kMaxGrpcErrorDescriptionSize, '\0');
-        library_->GetErrorMessage(vi, status, nidevice_grpc::kMaxGrpcErrorDescriptionSize, &description[0]);
-    }
-    return nidevice_grpc::ApiErrorAndDescriptionToStatus(status, description);
+  ViStatus error_code{};
```

Comment:
Nit: In the generated code there's a space before `{};` so I kept that style in the custom code I wrote. You've removed it here.

Reply:
Ah, I have my vs code to auto-format when I save based on our [clang format file](https://github.com/ni/grpc-device/blob/main/.clang-format). I reverted these formatting changes, though. 

We can update the .clang-format file and add something like `SpaceBeforeCpp11BracedList: true` to get the space before braces when auto-formatted. But we can do that some other time.